### PR TITLE
fix: Expand Contract button and Removing line

### PR DESCRIPTION
**Pull request by palbizu**
## Description

- Expand / Collapse button color changed when open (a variable was added to allow customization)
- Left lateral dotted line was removed and the padding was adjusted to that position.

### Testing
Visual test.

### Checklist:
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### Documentation
Make sure that you have documented corresponding changes in this repository or [hypertrace docs repo](https://github.com/hypertrace/hypertrace-docs-website) if required.

image

### CODE REVIEW

File: projects/components/src/expander/expander-toggle.component.ts [hunk 1]
```diff
@@ -27,6 +29,9 @@ export class ExpanderToggleComponent {
   @Input()
   public showTooltip: boolean = true;
 
+  @Input()
+  public colorChange: boolean = false;
```
Comment (anandtiwary) [inline projects/components/src/expander/expander-toggle.component.ts:33]: Do we need this as an input? Are there any place where we would want to use the old behavior?

Comment (palbizu) [reply projects/components/src/expander/expander-toggle.component.ts:33]: I added to make it more customizable. Do you think that I should remove it? We're are going to use it with the change in all ocurrences.

Comment (aaron-steinfeld) [reply projects/components/src/expander/expander-toggle.component.ts:33]: If we don't need it now, I'd remove it - it's much easier to add a new input later when we have a use case than remove it.

As a separate comment (not relevant if removing it) - the suffix `*Change` is generally reserved for outputs, as that's the angular framework's naming expectation for banana in box syntax (https://angular.io/guide/two-way-binding#how-two-way-binding-works)

Comment (palbizu) [reply projects/components/src/expander/expander-toggle.component.ts:33]: great, thanks for the advice I appreciate it

File: projects/distributed-tracing/src/shared/dashboard/widgets/trace-detail/trace-detail-widget-renderer.component.scss [hunk 2]
```diff
@@ -3,7 +3,7 @@
     flex: 1 1 auto;
     display: flex;
     flex-direction: row;
-    padding: 0 16px;
+    padding: 0 16px 0 0;
```
Comment (aaron-steinfeld) [inline projects/distributed-tracing/src/shared/dashboard/widgets/trace-detail/trace-detail-widget-renderer.component.scss:6]: nit: more readable just to specify `padding-right`, IMO

File: projects/components/src/expander/expander-toggle.component.ts (hunk 1 above)
Comment (aaron-steinfeld) [reply projects/components/src/expander/expander-toggle.component.ts:31]: This got removed in one commit, then added back in the next - was that intentional?

**Review by aaron-steinfeld — DISMISSED** [review]

File: projects/distributed-tracing/src/shared/dashboard/widgets/trace-detail/trace-detail-widget-renderer.component.scss (hunk 2 above)
Comment (palbizu) [reply projects/distributed-tracing/src/shared/dashboard/widgets/trace-detail/trace-detail-widget-renderer.component.scss:6]: I thought about that but I didn't have clear if the padding was stepping on something

Comment (aaron-steinfeld) [reply projects/distributed-tracing/src/shared/dashboard/widgets/trace-detail/trace-detail-widget-renderer.component.scss:6]: I'd hope not, but that's fine to leave it, just a tiny nit.

File: projects/components/src/expander/expander-toggle.component.ts (hunk 1 above)
Comment (palbizu) [reply projects/components/src/expander/expander-toggle.component.ts]: It should need to be removed, I added it by mistake.

File: projects/distributed-tracing/src/shared/dashboard/widgets/trace-detail/trace-detail-widget-renderer.component.scss (hunk 2 above)
Comment (palbizu) [reply projects/distributed-tracing/src/shared/dashboard/widgets/trace-detail/trace-detail-widget-renderer.component.scss]: I replaced it. Thanks!

**Review by aaron-steinfeld — APPROVED** [review]

**Review by anandtiwary — APPROVED** [review]

**Pull request merged**